src/components/tags/book-selection.cjsx

```diff
@@ -0,0 +1,39 @@
+React = require 'react'
+_ = require 'underscore'
+
+BOOKS =
```

Comment:
yeah, not really sure on that.  I expect the process of someone who's attempting to update the book lists to be: find tags/book component.  Discover it renders the shared book-selection component and then update the list.  Given your suggestion it'd be an additional bit of detective work to find the src/constants.json file.   

Ideally, I'd say that the BE would have an `/admin` page like Tutor that this could be set in, but we're not quite there yet.